package.json

```
@@ -6,6 +6,7 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@metabase/embedding-sdk-react": "^0.56.3",
```

Biome (2.1.2)

[error] The key @metabase/embedding-sdk-react was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)